dynacred/src/Data.ts

```diff
@@ -126,7 +128,7 @@ export class Data {
      * @param unrestricted whether the adminDarcID needs evolve_unrestricted
      */
     static async createFirstUser(bc: ByzCoinRPC, adminDarcID: InstanceID, adminKey: Scalar,
-                                 alias: string, unrestricted: boolean = false):
+                                 alias: string, unrestricted?: boolean):
```

Comment:
In this case, you supports three values for `unrestricted`, `true`, `false` and `undefined`. Is it wanted?

Reply:
premature @tharvik optimization...